container/container.go

```diff
@@ -122,6 +122,7 @@ type SecurityOptions struct {
 	AppArmorProfile string
 	SeccompProfile  string
 	NoNewPrivileges bool
+	WritableCgroups bool `json:"writable-cgroups,omitempty"`
```

Comment:
This json tag doesn't seem consistent with others

Reply:
how come?

Reply:
Other options in the struct do not have `json` tags

Reply:
oh oh, you said json tag, and my brain read variable name.
yes. @thaJeztah said about having the json tag, but to smooth it out, would mean changes on lines that have nothing to do with this PR ... so I didn't touch them.

Reply:
So, `json:"writable-cgroups,omitempty"` should be removed too?

Reply:
I'm ok with that. Since I've heard both for and against, I would welcome a tie-breaker from other maintainers.

Reply:
Sorry, I'm lagging behind on my notifications; yeah, so my comment was related to some situations whree we didn't define the casing to use for fields, and some `ID` field renamed (`Id -> ID`). Golang itself is (sigh) case-insensitive for JSON fields, so all seemed fine, but we broke some non-Go tools, so we now had to explicitly set `Id` after-the-fact to preserve the `Id` casing. So my thinking here was that it's better to be explicit instead of depending on Go's behavior. 

The `omitempty` was from a perspective that it's a new field, and adding the `omitempty` could possibly prevent container configs being updated on-disk during the "restore" cycle on daemon startup.

Reply:
Should I add a commit that also tags the other fields as well?
(that feels like venturing into ensuring and fixing marshalling issues
that are unrelated to this PR...)